docs/conf.py

```diff
@@ -36,7 +36,7 @@
 import sys
 from typing import Dict
 
-import airflow
```

Comment:
(Btw this will still import/load/parse `airflow/__init__.py`, it just won't make it "in scope")

Reply:
Yep. I learned it :).

Reply:
`airflow.settings` is not used in this file. Do we should import `version` from `airflow.version`?

Reply:
Seems just `import airflow` does the work nicely.